tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/skeleton.ts

```diff
@@ -0,0 +1,33 @@
+import { Species } from "./base";
+
+const Skeleton: Species = {
+  description: "A rattling skeleton! They descend upon Space Station Thirteen \
```

Comment:
We call it 13 in the nuke ops description, but doesn't matter so much.

Reply:
UNITY